docs/getting_started/creating_flyte_project.md

```diff
@@ -45,7 +45,7 @@ git init  # initialize a git repository
 If you examine `my_project`, you'll see the following file structure:
 
 ```{code-block} bash
-my_project
+tree my_project
```

Comment:
this code snippet isn't actually a invocation of `tree`, it's just to show the directory structure... consider reverting, or adding an explicit `tree my_project` above this code cell.

Reply:
Done :)